sklearn/ensemble/tests/test_bagging.py

```diff
@@ -748,3 +749,51 @@ def test_set_oob_score_label_encoding():
     x3 = BaggingClassifier(oob_score=True,
                            random_state=random_state).fit(X, Y3).oob_score_
     assert_equal([x1, x2], [x3, x3])
+
+
+def test_bagging_pipeline_with_sparse_inputs():
+    # Check that BaggingRegressor can accept sparse pipelines inputs
+    X = [
+        [1, 3, 5],
+        [2, None, 6],
```

Comment:
I think you want np.nan rather than None?

Reply:
TBH, I generally test against both as I'm not entirely sure what happens under the covers. I know I can initialize a Pandas DataFrame either way.

Reply:
Added separate imputers/inputs for None/np.nan, np.inf, np.NINF.